frontend/www/js/omegaup/components/common/Typeahead.vue

```diff
@@ -32,12 +32,15 @@ import { types } from '../../api_types';
 })
 export default class Typeahead extends Vue {
   @Prop() existingOptions!: types.ListItem[];
+  @Prop({ default: () => [] }) options!: types.ListItem[];
   @Prop({ default: 3 }) activationThreshold!: number;
   @Prop({ default: 5 }) maxResults!: number;
   @Prop({ default: null }) value!: null | string;
+  @Prop({ default: true }) onlyExistingTags!: boolean;
+  @Prop({ default: T.typeaheadSearchPlaceholder }) placeholder!: boolean;
```

Comment:
```suggestion
  @Prop({ default: T.typeaheadSearchPlaceholder }) placeholder!: string;
```

esto es una cadena, no un booleano

Reply:
Listo